src/SoapstoneLib/SoapstoneLib/SoapstoneLib.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="Google.Protobuf" Version="3.25.2" />
     <PackageReference Include="Grpc.AspNetCore" Version="2.60.0" />
     <PackageReference Include="Grpc.Net.Client" Version="2.60.0" />
+    <PackageReference Include="ZstdNet" Version="1.4.5" />
```

Comment:
soapstone is requiring it?

Reply:
It's the new compression for regulation.bin